src/commands/push.rs

```diff
@@ -56,9 +56,10 @@ impl PushArgs {
                 ).await
             };
 
-            if result.is_err() {
-                error!("Failed to push locale {:?} from {}", locale, output_path);
-            }
+            match result {
+                Err(e) => return Err(WordsmithError::Push { file: output_path, locale, cause: Box::new(e) }),
+                Ok(_) => {},
+            };
```

Comment:
You should be able to do this with `if let` right?
```suggestion
            if let Err(e) = result {
             return Err(WordsmithError::Push { file: output_path, locale, cause: Box::new(e) })
            };
```